src/chat/model_selector.rs

```diff
@@ -1,5 +1,5 @@
 use crate::{
```

Comment:
Changes in this file are about not showing the model in the selector as "loading" if it's not the one actually being loaded.

For example, if the chat has an associated model A, and you used `@` to invoke another model temporally.